common/common.go

```diff
@@ -54,6 +54,7 @@ const (
 	DefaultRegistryHostName       = "registry.cn-qingdao.aliyuncs.com"
 	DefaultRegistryAuthDir        = "/root/.docker/config.json"
 	KubeAdminConf                 = "/etc/kubernetes/admin.conf"
+	K0sAdminConf                  = "/var/lib/k0s/pki/admin.conf"
```

Comment:
move `K0sAdminConf` to k0s runtime common file ？it is a built-in constant of k0s.